activerecord/lib/active_record/errors.rb

```diff
@@ -94,13 +94,21 @@ def initialize(message = nil, record = nil)
 
   # Superclass for all database execution errors.
   #
-  # Wraps the underlying database error as +original_exception+.
+  # Wraps the underlying database error as +cause+.
   class StatementInvalid < ActiveRecordError
-    attr_reader :original_exception
 
     def initialize(message = nil, original_exception = nil)
-      @original_exception = original_exception
-      super(message)
+      if original_exception
```

Comment:
removed in https://github.com/rails/rails/commit/bc6c5df4699d3f6b4a61dd12328f9e0f1bd6cf46 ( noting because I was just starting to use original_exception in my Rails 4.2. app's logging when I noticed it swallowing the original exception.... )  `¯\_(ツ)_/¯`